t/03-proxy_wasm/hfuncs/131-proxy_dispatch_http_timeouts.t

```diff
@@ -97,6 +98,7 @@ qr/(\[error\]|Uncaught RuntimeError|\s+).*?dispatch failed: tcp socket - timed o
 
 
 === TEST 4: proxy_wasm - dispatch_http_call() write timeout
+--- skip_eval: 4: $::osname =~ m/darwin/
```

Comment:
Should we also add the reason for skipping these two tests on macos?

Reply:
Good call.